deploy/docker-compose.yml

```diff
@@ -128,32 +131,6 @@ services:
     volumes:
       - ./state/webroot:/usr/share/nginx/html/
 
-  cacher:
```

Comment:
So we totally get ride of cacher from tinkerbell deployment. But do we still deploy cacher from other place? I can still see Hegel still interact with cacher in the other PR.

Reply:
yeah, cacher is still used in other places, just not in tinkerbell